### PR TITLE
Remove db work mem

### PR DESCRIPTION
**Pull request by irisdianauy**
This removes setting database working memory for some postgres routines, which are causing the out-of-memory errors seen in single-cell experiment loading.  It is also now rebasing onto develop from fa23a2ac8cd2c658b6050379a007f96809d81e06, and bringing changes from the latter onto the main branch.

### CODE REVIEW

**Comment by irisdianauy** [comment]
This has been rebased onto develop

**Comment by anilthanki** [comment]
Whats the reason for sourcing `common_routines.sh`

**Comment by irisdianauy** [comment]
> Whats the reason for sourcing `common_routines.sh`

This was part of rebasing this branch onto develop, which had to be done because the commit we use in the atlas-prod db-scxa submodule has diverged from db-scxa develop.  If you compare fa23a2ac8cd2c658b6050379a007f96809d81e06 and develop, you will see that a script has been sourced in fa23a2ac8cd2c658b6050379a007f96809d81e06 but that script has since been replaced by `common_routines.sh` in develop.

**Review by anilthanki — APPROVED** [review]
LGTM

File: bin/load_db_scxa_dimred.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ EXP_ID=${EXP_ID:-$2}
 DIMRED_TYPE=${DIMRED_TYPE:-$3}
 DIMRED_FILE_PATH=${DIMRED_FILE_PATH:-$4}
 DIMRED_PARAM_JSON=${DIMRED_PARAM_JSON:-$5}
+#SCRATCH_DIR=${SCRATCH_DIR:-"$(dirname ${DIMRED_FILE_PATH})"}
```
Comment (ke4) [inline bin/load_db_scxa_dimred.sh]: Could you please remove this commented line?
Thanks

File: bin/load_exp_design.sh [hunk 2]
```diff
@@ -3,6 +3,9 @@ set -e
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}" )" &> /dev/null && pwd )
 source "${SCRIPT_DIR}/common_routines.sh"
 
+scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]:-$0}" )" && pwd )
```
Comment (ke4) [inline bin/load_exp_design.sh]: Is this replacing line 3-4?
If yes, then shall we remove those line, please?
Thanks

**Review by ke4 — CHANGES_REQUESTED** [review]
Good work!
I added 2 small comments/suggestions.
Could you check them please?
Thanks

File: bin/load_exp_design.sh (hunk 2 above)
Comment (irisdianauy) [reply bin/load_exp_design.sh]: Thanks, done in a71412ddbbad1d7721d658a277ee9257e3efb576

File: bin/load_db_scxa_dimred.sh (hunk 1 above)
Comment (irisdianauy) [reply bin/load_db_scxa_dimred.sh]: Done in a71412ddbbad1d7721d658a277ee9257e3efb576

**Review by ke4 — APPROVED** [review]
Great and quick work!
Thanks for applying my suggestions!
LGTM and approved

**Comment by irisdianauy** [comment]
Thanks @ke4 and @anilthanki!

**Pull request merged**